config.py

```diff
@@ -5,7 +5,7 @@ class DevConfig(object):
     SQLALCHEMY_ECHO = True
 
 class TestConfig(object):
-    DEBUG = True
+    DEBUG = False
```

Comment:
@ethanschreur why do we have to set debugging in the test config ?

Reply:
The CICD would not run tests with multiple setup functions. This change solved that problem.